drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -143,13 +143,13 @@
         <div class="row">
             <label class="h4">
                 <span>{{i18n "Success feedback"}}</span>
-                <textarea class="success-feedback">{{ feedback.correct }}</textarea>
+                <textarea class="success-feedback">{{i18n feedback.correct}}</textarea>
             </label>
         </div>
         <div class="row">
             <label class="h4">
                 <span>{{i18n "Error feedback"}}</span>
-                <textarea class="error-feedback">{{ feedback.incorrect }}</textarea>
+                <textarea class="error-feedback">{{i18n feedback.incorrect}}</textarea>
```

Comment:
I'm not sure what's the template tag `i18n` here, and why we are using it instead of `trans`.

Reply:
Now that's a good question, I don't have a lot of context here. My guess is that it's coming from the i18n service [here](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/172/files#diff-81cf2d60a06d8f43d0df802d9aad9417R376) and [here](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/172/files#diff-81cf2d60a06d8f43d0df802d9aad9417R588)

It's being extracted correctly though:
```
#: templates/html/js_templates.html
msgid "Error feedback"
msgstr ""
```